src/__test__/redux/actions/pipelines/runGem2s.test.js

```diff
@@ -97,4 +101,21 @@ describe('runGem2s action', () => {
 
     expect(fetchMock).toHaveBeenCalled();
   });
+
+  it('Works well with api v2', async () => {
```

Comment:
I am considering that we will need to revisit this once we migrate to v2 to remove the lines `config.currentApiVersion = api.V2;` and potentially renaming the tests. Is there a way to avoid that? maybe instead of adding an extra test case and extra file? like `runGem2s.v2.test.js` so when we migrate we only need to delete the other file without changing any code (we could potentially rename this one to `runGem2s.v1.test.js` to make it easier to find and remove them when we are done)

Reply:
Many of the tests can be reused once we switch to using v2 and in many cases the changes done in the action creator are not many, that's why I feel like that refactor might not be worth the amount of work it involves (since we will also need work to undo it).